src/OpenTelemetry.Extensions.Serilog/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenTelemetry.Extensions.Serilog
+
+This project contains a [Serilog](https://github.com/serilog/)
+[sink](https://github.com/serilog/serilog/wiki/Configuration-Basics#sinks) for
+writing log messages to OpenTelemetry.
+
+## Usage Example
+
+```csharp
+// Step 1: Configure OpenTelemetryLoggerProvider...
+var openTelemetryLoggerProvider = new OpenTelemetryLoggerProvider(options =>
+{
+    options
+        .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService("MyService"))
+        .AddConsoleExporter();
+});
+
+// Step 2: Register OpenTelemetry sink with Serilog...
+Log.Logger = new LoggerConfiguration()
+    .WriteTo.OpenTelemetry(openTelemetryLoggerProvider, disposeProvider: true)
```

Comment:
```suggestion
    .WriteTo.OpenTelemetry(openTelemetryLoggerProvider)
```

Reply:
I did this in the example program and the README. My thinking was being explicit gives the users a hint to pay attention to that parameter.